src/main/java/me/botsko/prism/listeners/PrismEntityEvents.java

```diff
@@ -759,10 +760,17 @@ public void onEntityBlockForm(final EntityBlockFormEvent event) {
         final Block block = event.getBlock();
         final Location loc = block.getLocation();
         final BlockState newState = event.getNewState();
-        final String entity = event.getEntity().getType().name().toLowerCase();
 
-        RecordingQueue.addToQueue(ActionFactory.createBlockChange("entity-form", loc, block.getType(),
-                block.getBlockData(), newState.getType(), newState.getBlockData(), entity));
+        if (event.getEntity() instanceof Player) {
+            final OfflinePlayer player = (OfflinePlayer)event.getEntity();
+            RecordingQueue.addToQueue(ActionFactory.createBlockChange("entity-form", loc, block.getType(),
+                    block.getBlockData(), newState.getType(), newState.getBlockData(), player));
+        }
```

Comment:
Codacy found an issue: ['}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally).](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6131844)